### PR TITLE
#7.  Upload attendance doesn’t show “Day” dropdown. Feature should be…

### PR DESCRIPTION
**Pull request by BhagyashreeBhoyar**
… that user is able to upload attendance data for any specific day.

### CODE REVIEW

File: src/components/ColumnConfig.jsx [hunk 1]
```diff
@@ -62,21 +62,21 @@ class ColumnConfig extends Component {
    */
   renderColumnOptions = () => {
     let totalColumnCollection = [];
-    let tenColumnOptions = [];
+    let columnListChunks = [];
```
Comment (pepcus-pratik) [inline src/components/ColumnConfig.jsx]: Much better. Good.

File: src/components/UploadStudentsAttendanceFile.jsx [hunk 2]
```diff
@@ -47,6 +49,17 @@ class UploadStudentsAttendanceFile extends Component {
     this.state = {
       attendanceFile: null,
       isUploadStudentsAttendanceFileModal: false,
+      selectedDay: '',
+      days: [
```
Comment (pepcus-pratik) [inline src/components/UploadStudentsAttendanceFile.jsx]: No need to keep this in the component state.

File: src/components/UploadStudentsAttendanceFile.jsx [hunk 3]
```diff
@@ -132,6 +148,34 @@ class UploadStudentsAttendanceFile extends Component {
     }
     return null;
   }
+  /**
+   * addOptions method return options of drop down list
+   * of days
+   * @return {ReactComponent}
+   */
+  addOptions() {
+    const Options = ({ day }) => (
```
Comment (pepcus-pratik) [inline src/components/UploadStudentsAttendanceFile.jsx]: We should not create components like this. Either create a separate component in a the same/separate file or just write the JSX code directly.

Comment (BhagyashreeBhoyar) [reply src/components/UploadStudentsAttendanceFile.jsx]: I have refactored it.

File: src/components/UploadStudentsAttendanceFile.jsx (hunk 2 above)
Comment (BhagyashreeBhoyar) [reply src/components/UploadStudentsAttendanceFile.jsx]: Done.

**Pull request merged**